CHANGELOG.md

```diff
@@ -1,9 +1,21 @@
 # Postman Collection SDK Changelog
 
 #### Unreleased
+* :tada: Get and set objects in variables with `json` variable type
+```js
+var variable = new Variable({ type: 'json' });
+
+// set objects as values for `json types`
+variable.set({ version: 'v1' });
```

Comment:
the 2nd arg needs to be `json` --- isn't it?

Reply:
`variable.set` only takes the value as a parameter. `type` is managed separately(at construction of using `valueType`)

Reply:
👍🏼